Doc/c-api/sequence.rst

```diff
@@ -105,6 +105,15 @@ Sequence Protocol
    equivalent to the Python expression ``value in o``.
 
 
+.. c:function:: int PySequence_In(PyObject *o, PyObject *value)
+
+   Alias for :c:func:`PySequence_Contains`.
+
+   .. deprecated:: 3.14
```

Comment:
We have this fancy thing these days:
```suggestion
   .. deprecated:: next

```

Reply:
We do, and it makes backporting easier, but backporting a deprecation doesn't make much sense :)